### DISCUSSION TITLE
Improve standard Humidifier Card

### DISCUSSION THREAD

**Discussion by lakshinskij** [home-assistant/frontend#9780] [Dashboard]
Hello people. I want to talk about the standard Humidifier Card. https://www.home-assistant.io/lovelace/humidifier/
A year ago I bought my first Xiaomi humidifier.
I put a standard humidifier card in Lovelace UI, and I did not like it (((
During the year I tried a lot of different cards written by good people in the community.
But unfortunately I did not find anything suitable. Something is not beautiful, something is not functional ...
I returned to the standard card. 
There is a huge request to our dear "parents" Home Assistant.
Be so kind as to refine this card, please.
It would be great if it looked like a climate card. Here's a straightforward copy and paste.
I would like the real humidity to be displayed in the center, and the target below, as on the climate card.
I would like all the entities that Home Assistant finds in the connected humidifier to be available below.
That would be just great !!!!
Now the humidifier card is beautiful, but not convenient (((
with great respect Dmitry.

**Comment by spacegaier** [comment]
@matthiasdebaat Showing the current and target humidity values sounds reasonable to me, since that is what we do with the thermostat card. What do you think?

@lakshinskij

> I would like all the entities that Home Assistant finds in the connected humidifier to be available below.

What do you mean by that? I don't have any humidifiers in HA, so I am not sure.

**Reply by lakshinskij** [reply]
Yes, but there is only a target humidity in the humidifier card. And there is not even an on / off button. When a humidifier is added to the system, new entities are discovered. After the humidifier entitie, we have switch.humidifier_buzzer, switch.humidifier_child_lock, sensor.humidifier_water_level, binary_sensor.humidifier_water_tank, sensor.humidifier_humidity, sensor.humidifier_temperature
So, from the card I can only set the target humidity. To change the power or turn on / off, I need to press three points, and on and on. It is very uncomfortable. Everything should be before your eyes on the card. Actually, as it is done in the thermostat card. I apologize for my english. I am from Kiev, Ukraine

**Reply by matthiasdebaat** [reply]
> @matthiasdebaat Showing the current and target humidity values sounds reasonable to me, since that is what we do with the thermostat card. What do you think?

I agree, good suggestion @lakshinskij. Let's make the `humidity` and `thermostat` consistent.
